Doc/library/functions.rst

```diff
@@ -1371,7 +1371,7 @@ are always available.  They are listed here in alphabetical order.
       supported.
 
 
-.. function:: print(*objects, sep=' ', end='\\n', file=sys.stdout, flush=False)
+.. function:: print(value, ..., sep=' ', end='\n', file=sys.stdout, flush=False)
```

Comment:
The documentation for the `*args` argument is more correct as it currently stands; the output of `help()` doesn't look great here. Good catch on the typo in the `end` argument, though.

```suggestion
.. function:: print(*objects, sep=' ', end='\n', file=sys.stdout, flush=False)
```